tests/tinypages/index.rst

```diff
@@ -1,5 +1,5 @@
-Welcome to tinypages's documentation!
-=====================================
+Welcome to Tinypages's Documentation
+====================================
```

Comment:
Is it grammatical to have `...s's`? I was always taught that we omit the `s` after the apostrophe in this case. Or perhaps sidestep the whole issue with
```suggestion
Welcome to the Documentation of Tinypages
=========================================
```